src/components/Explore/SearchBar.tsx

```diff
@@ -50,18 +46,22 @@ const SearchInput = styled.input<{ expanded: boolean }>`
     background-size: ${ICON_SIZE} ${ICON_SIZE};
     cursor: pointer;
   }
+
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    width: 100%;
+  }
 `
 
 export default function SearchBar() {
   const [filterString, setFilterString] = useAtom(filterStringAtom)
-  const [isExpanded, setExpanded] = useState(false)
+  const [isExpanded, setExpanded] = useState(filterString.length !== 0)
```

Comment:
I asked in eng-web-explore haha

Reply:
I think it should reset too I guess since the results would probably not be a list of tokens, but rather, a user would be searching for a specific token?